onnxruntime/core/providers/cpu/nn/conv_transpose_attributes.h

```diff
@@ -176,6 +176,8 @@ struct ConvTransposeAttributes : public ConvAttributes {
 
   void DistributePadding(AutoPadType pad_type, const int64_t& total_pad,
                          int64_t& pad_head, int64_t& pad_tail) const {
+    // TODO(jcwchen): #9740 ORT 1.13 will correct the logic by switching them to meet ONNX spec
+    LOGS_DEFAULT(WARNING) << "The pad result for SAME_UPPER and SAME_LOWER will be corrected in next ORT 1.13 release.";
```

Comment:
Nit: Will this warning be more explicit about the intent ?

"The existing bug in the padding distribution for auto_pad type SAME_UPPER/SAME_LOWER will be fixed in next ORT 1.13 release and hence the results of ConvTranspose operator using the above auto_pad type(s) will be different."

Or something similar that explicitly calls out that ConvTranspose will yield different results for SAME_UPPER/SAME_LOWER rather than just stop with "corrected" which does not state the net result of that.

Reply:
Yes definitely more explicit warning is better. I have updated it by your first one. Thank you for the detailed message!